components/automate-ui/src/app/modules/team/team-details/team-details.component.ts

```diff
@@ -121,9 +114,9 @@ export class TeamDetailsComponent implements OnInit, OnDestroy {
         (uStatus === EntityStatus.loading) ||
         (usersStatus === EntityStatus.loading);
       if (this.isLoadingTeam) {
-        this.updateNameForm.controls['name'].disable();
+        this.updateForm.controls['name'].disable();
```

Comment:
I believe in other places you've rewritten those, too?
```suggestion
        this.updateForm.controls.name.disable();
```
but I may be mistaken

Reply:
Yes, thank you!

Reply:
We've got a bit of a mixture of styles across the app (not a shock of course). We also use `.get` in various places:

https://angular.io/api/forms/AbstractControl#get

```ts
this.updateForm.get('name').disable();
```

`.get()` is nice for drilling down into nested controls but other than that not sure of any other pros/cons between the various ways to access form controls.

Reply:
Nice! Did not know that one; thanks @scottopherson .